core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div class="conversation-skin-back-to-collection-container">
```

Comment:
One way to do this is to look at the browser history using [referrer](https://www.w3schools.com/jsref/prop_doc_referrer.asp).